src/rgw/driver/dbstore/common/dbstore.h

```diff
@@ -1628,6 +1633,7 @@ class DB {
     // "<bucketname>_<objname>_<objinstance>_<multipart-part-str>_<partnum>"
     static constexpr std::string_view raw_obj_oid = "{0}_{1}_{2}_{3}_{4}";
 
+    // XXX [Matt] I think these are actual part numbers, and that we don't need stripe here
```

Comment:
This is probably stripe_num too.. Also InitializeParamsfromRawObj() needs update.

Reply:
ok, will check

Reply:
yep